modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -279,6 +279,13 @@ export class Wallet implements IWallet {
     return this._wallet.receiveAddress.address;
   }
 
+  /**
+   * Get a list of tokens and their balances from a wallet
+   */
+  public tokens(): Record<string, any>[] | undefined {
```

Comment:
To make sure, is this a convenience function over `toJSON` or is there another intent as well?

Reply:
it's a getter for the tokens because they're not accessible otherwise

Reply:
Same way the functions above operate pretty much :)

Reply:
But I can also do that via `toJson`, yes?

Reply:
oh shoot, yeah you can i see, hadnt seen that in the wallet class before, so i guess it is entirely just a convenience function then

Reply:
happy to change that if needed!

Reply:
Yeah lets use what we have for now. Thanks!

Reply:
You can still leave the tokens type on there. Will help for clarity and documentation